library/src/main/java/com/novoda/downloadmanager/LiteDownloadMigrationService.java

```diff
@@ -0,0 +1,124 @@
+package com.novoda.downloadmanager;
+
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.IBinder;
+import android.os.SystemClock;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class LiteDownloadMigrationService extends Service {
+
+    private Migrator v1ToV2Migrator;
+    private ExecutorService executor;
+    private IBinder binder;
+    private Migrator.Callback migrationCallback;
+    private String updateMessage;
```

Comment:
Need to store the previous value so that when binding we get the latest update message initially.